src/Ryujinx.Graphics.Nvdec.Vp9/LoopFilter.cs

```diff
@@ -78,8 +76,7 @@ internal static class LoopFilter
         //  00000000
         //  00000000
         //  00000000
-        private static readonly ulong[] LeftPredictionMask = new ulong[]
-        {
+        private static readonly ulong[] _leftPredictionMask = {
             0x0000000000000001UL,  // BLOCK_4X4,
```

Comment:
```suggestion
            0x0000000000000001UL, // BLOCK_4X4,
```
Same for other comments